playbooks/roles/tower/tasks/bootstrap.yml

```diff
@@ -2,6 +2,19 @@
 # Authenticate with tower
 - include_tasks: authenticate_tower.yml
 
+- name: Upload license file "https://{{ tower_host }}"
+  uri:
+    url: "https://{{ tower_host }}/api/v2/config/"
+    return_content: yes
+    user: "{{ tower_username }}"
+    password: "{{ tower_password }}"
+    method: POST
+    force_basic_auth: yes
+    body_format: json
+    body: '{{ tower_license }}'
+    validate_certs: '{{ tower_verify_ssl }}'
+  when: tower_license != ''
```

Comment:
@laurafitzgerald You were right by having `when: tower_license and tower_license != '' ` - apologies! While testing I had an environment that didn't specify a license var and it failed. There will be other cases like this so we should re-instate what you originally had 😄

Reply:
Actually, as it turns out the original `when` statement won't work either as it still expects an `<env>_tower_license` variable to exist. Not sure what we can do here really except state clearly in our docs that this is a mandatory variable in each cred repo - in any case, you can leave everything as is!

Reply:
Yes, I think the case is the same here as if the username or password were left out. The thing would fail somehow. So if we expect them to provide that I think it's reasonable to expect that to be included even if it's empty.